ee/clickhouse/sql/cohort.py

```diff
@@ -87,6 +95,14 @@
 HAVING sum(sign) > 0
 """
 
+GET_COHORTS_BY_PERSON_UUID_VERSIONED = """
+SELECT cohort_id
```

Comment:
Nit: do we want a `distinct cohort_id` here? Otherwise, I think we could end up in a state where this is returning duplicate cohort_ids because of inserts not being atomic (e.g. the recalculate is actively happening, or part of the recalculate failed)